LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2022 Babbel GmbH
```

Comment:
@awendt I set it to 2022 because that was when the repository was setup. I would suggest not specifying a range of years because then we would need to update regularly.

So, 2022 or 2023?